cmd/transport_security_state_static_generate/transport_security_state_static_generate.go

```diff
@@ -62,13 +62,15 @@ type pinset struct {
 }
 
 type hsts struct {
-	Name                 string `json:"name"`
-	Subdomains           bool   `json:"include_subdomains"`
-	SubdomainsForPinning bool   `json:"include_subdomains_for_pinning"`
-	Mode                 string `json:"mode"`
-	Pins                 string `json:"pins"`
-	ExpectCT             bool   `json:"expect_ct"`
-	ExpectCTReportURI    string `json:"expect_ct_report_uri"`
+	Name                  string `json:"name"`
+	Subdomains            bool   `json:"include_subdomains"`
```

Comment:
Alright, I'm working on the relevant change within Chrome. I'll submit that CL later today, and we can coordinate patches.